### PR TITLE
feat(show tips): 用户输入错误之后显示音标，输入正确之后恢复用户设置的默认状态 fixed #55

### LINKED ISSUE #55: 默写拼错时希望能加上音标提示
鉴于同义词的问题，希望在默写模式下写错时加上音标提示，感谢。

### PR DESCRIPTION
**Pull request by sdu-gyf**

### CODE REVIEW

File: src/components/Word/index.tsx [hunk 1]
```diff
@@ -113,6 +115,7 @@ export type WordProps = {
   onFinish: Function
   isStart: boolean
   wordVisible: boolean
+  switchTips: any
```
Comment (chengluyu) [inline src/components/Word/index.tsx:118]: Why `any`?

File: src/pages/Typing/Switcher/index.tsx [hunk 2]
```diff
@@ -6,9 +6,10 @@ import { useHotkeys } from 'react-hotkeys-hook'
 export type SwitcherPropsType = {
   state: SwitcherStateType
   dispatch: SwitcherDispatchType
+  changeUserPhoneticState: any
```
Comment (chengluyu) [inline src/pages/Typing/Switcher/index.tsx:9]: Why `any`?

**Review by chengluyu — CHANGES_REQUESTED** [review]
Please remove `any` types.

File: src/components/Word/index.tsx (hunk 1 above)
Comment (sdu-gyf) [reply src/components/Word/index.tsx]: sorry, I'll replace it with correct type later

File: src/pages/Typing/Switcher/index.tsx (hunk 2 above)
Comment (sdu-gyf) [reply src/pages/Typing/Switcher/index.tsx]: sorry, I'll replace it with correct type later

**Review by chengluyu — APPROVED** [review]

**Pull request merged**